Lite Docs and Example Improvements (#10303)
PyTorchLightning/pytorch-lightning: pl_examples/basic_examples/README.md

```bash
# CPU
python mnist_examples/image_classifier_5_lightning_datamodule.py

# GPUs (any number)
python mnist_examples/image_classifier_5_lightning_datamodule.py --trainer.gpus 2

# Distributed Data Parallel (DDP)
python mnist_examples/image_classifier_5_lightning_datamodule.py --trainer.gpus 2 --trainer.strategy 'ddp'
```

______________________________________________________________________

#### AutoEncoder

> This script shows you to implement an CNN auto-encoder.

Reviewer: ```suggestion
This script shows you how to implement a CNN auto-encoder.
```

Author: Done !

```bash
# CPU
python autoencoder.py

# GPUs (any number)
python autoencoder.py --trainer.gpus 2

# Distributed Data Parallel (DDP)
python autoencoder.py --trainer.gpus 2 --trainer.strategy 'ddp'
```

______________________________________________________________________

#### Backbone Image Classifier

> This script shows you to implement a `LightningModule` as a `System`.

Reviewer: ```suggestion
This script shows you how to implement a `LightningModule` as a `System`.
```
Is "System" meant to be formatted?

Author: Yes, I would say it is a special term.

> A System describes a `LightningModule` which takes a single a `torch.nn.Module` which makes

Reviewer: ```suggestion
A system describes a `LightningModule` which takes a single `torch.nn.Module` which makes
```
I don't think we have formalized "System" somewhere. It's not a name, right?

Author: Removed. But I think we should actually formalized the term. The community has been using it for a while now.

> exporting to producion simpler.

Reviewer: ```suggestion
makes exporting to producion simpler.
```

```bash
# CPU
python backbone_image_classifier.py

# GPUs (any number)
python backbone_image_classifier.py --trainer.gpus 2

# Distributed Data Parallel (DDP)
python backbone_image_classifier.py --trainer.gpus 2 --trainer.strategy 'ddp'
```

______________________________________________________________________

#### PyTorch Profiler

> This script shows you to activate the [PyTorch Profiler](https://github.com/pytorch/kineto) with Lightning.

Reviewer: ```suggestion
This script shows you how to activate the [PyTorch Profiler](https://github.com/pytorch/kineto) with Lightning.
```
